canvasapi/assignment.py

```diff
@@ -70,6 +88,54 @@ def get_gradeable_students(self, **kwargs):
             self._requester,
             'GET',
             'courses/{}/assignments/{}/gradeable_students'.format(self.course_id, self.id),
+            {'course_id': self.course_id},
+            _kwargs=combine_kwargs(**kwargs)
+        )
+
+    def get_override(self, override, **kwargs):
+        """
+        Get the single override with the given override id.
+
+        :calls: `GET /api/v1/courses/:course_id/assignments/:assignment_id/overrides/:id
+        <https://canvas.instructure.com/doc/api/assignments.html#method.assignment_overrides.show>`_
+
+        :param override: The obejct or ID of the override to get
```

Comment:
"The obejct" -> "The object"

Also, above, in L97:

"Get the single override with the given override id." -> Get **a** single override with the...